models/experimental/functional_unet/tests/test_unet_trace.py

```diff
@@ -91,7 +91,7 @@ def test_unet_trace(
     l1_input_tensor = ttnn.allocate_tensor_on_device(
         shape, dtype, layout, device, ttnn_model.input_sharded_memory_config
     )
-    assert input_trace_addr == l1_input_tensor.buffer_address()
+    # assert input_trace_addr == l1_input_tensor.buffer_address()
```

Comment:
@tt-aho I had to disable these asserts in order for tests to pass.
This might be a bit specific as conv2d in the model is set to deallocate input tensor.
Is this assert expected to pass in these circumstances?

Reply:
@pavlejosipovic I tried running this locally on an N150 and the asserts passed. Were you seeing the asserts fail on a different card like N300?

Reply:
Yes these tests are scheduled on N300 on nightly and it was failing there

Reply:
https://github.com/tenstorrent/tt-metal/actions/runs/12392722175/job/34592691327
This is one example

Reply:
Might be the wrong link? I think that run has the asserts commented out

Reply:
I wasn't able to repro locally on an N300 (from a t3k). Were you able to repro this locally?

Reply:
I haven't tried I'll give it a shot

Reply:
Thanks. I'd say this isn't a blocker but requires further investigation as to why output is correct when the input address to the model during trace is not the same as during capture (since that would normally be expected to produce incorrect output since we'd read from a stale address), or if you aren't able to repro locally either what the difference is on CI.